### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by JosePabloPQ** [cwendt94/espn-api#347] [Q&A] [answered]
I'm new to Python and working with GitHub and everything. I'm having trouble just initializing the Football side of things, I get some errors (4 to be precise). I did the following:

I installed the espn_api package

from espn_api.football import League
league = League(my_league_id, 2021, my_espn_s2, my_swid, my_username, my_password)

If you have any feedback I would greatly appreciate it.

Thank you and I appreciate your time in advance.

1. socket.gaierror: [Errno 8] nodename nor servname provided, or not known

2. Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known

3. HTTPSConnectionPool(host='ha.registerdisney.go.com', port=443): Max retries exceeded with url: /jgc/v5/client/ESPN-FANTASYLM-PROD/guest/login?langPref=en-US (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103508b50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))

4. requests.exceptions.ConnectionError: HTTPSConnectionPool(host='ha.registerdisney.go.com', port=443): Max retries exceeded with url: /jgc/v5/client/ESPN-FANTASYLM-PROD/guest/login?langPref=en-US (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103508b50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))

**Comment by cwendt94** [comment, marked as answer]
Hi @JosePabloPQ, try initializing the league without `username` and `password`. I have been meaning to remove those fields because it no longer works. ESPN changed how they authenticate username and password.

**Reply by JosePabloPQ** [reply]
Wow! Thank you very much Christian! I appreciate your help a lot! That worked perfectly.

**Reply by cwendt94** [reply]
No problem! Glad it is working!

**Comment by jackwalkernz** [comment]
Hey @cwendt94 . I am getting a League ID doesn't exist error. I have gotten my league ID from the URL for the league and have converted it to an integer, but it's not returning anything. Any suggestions?

**Reply by jackwalkernz** [reply]
Don't worry, league had to be 2023

**Reply by cwendt94** [reply]
Awesome, so everything is good! 😄

**Reply by jackwalkernz** [reply]
Yeah, everything appears to be all good! It's a very handy API. I've incorporated it with the nba_api project and am building out a model that runs my league teams for me!
